oceannavigator/frontend/src/components/lib/SelectBox.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { FormGroup, ControlLabel, FormControl } from "react-bootstrap";
+import PropTypes from "prop-types";
+const fastEqual = require('fast-deep-equal/es6/react');
```

Comment:
Hate adding yet another dependency but this provides the fastest deep object comparison function available. This is needed to tell React whether or not this component should be rendered. This will help keep the UI running quickly.